README.md

```diff
@@ -2079,9 +2061,6 @@ terratest:
     upgradeKubernetesVersion: ""
     controlPlaneConcurrencyValue: "15%"
     workerConcurrencyValue: "20%"
-    controlPlaneUnavailableValue: "1"
-    workerUnavailableValue: "10%"
```

Comment:
why rip these out?

Reply:
These specifically are for RKE1 downstream clusters. However, we since removed RKE1 from tfp-automation, so no need to have them in the readme.